step03.md

```diff
@@ -84,7 +84,9 @@ $ node server.js
 
 And if you are using cloud9:
 
-Simply click the `Run` button on the top menu.
+- Select the file `server.js` in the workspace folder tree.
+- Right click the `Run` button on the top menu and select `New Run Configuration` from the menu. This will open the Run Panel at the bottom of your workspace.
+- Type `node server.js` in the Command text editor and press Enter.
```

Comment:
@lauragieco  either this. Or let's make it run from the terminal the normal way. I reckon this mixing of the run button and code may confuse people.